### PR TITLE
[jsk_perception/bounding_box_to_rect] add rosparam approximate sync and queue_size

### PR DESCRIPTION
**Pull request by knorth55**

### CODE REVIEW

File: jsk_perception/src/bounding_box_to_rect.cpp [hunk 1]
```diff
@@ -45,6 +45,8 @@ namespace jsk_perception
   {
     DiagnosticNodelet::onInit();
     tf_listener_ = jsk_recognition_utils::TfListenerSingleton::getInstance();
+    pnh_->param("queue_size", queue_size_,100);
+    pnh_->param("approximate_sync",approximate_sync_,false);
```
Comment (wkentaro) [inline jsk_perception/src/bounding_box_to_rect.cpp]: Please fix style (space after comma)

**Comment by wkentaro** [comment]
Please update documentation.

**Comment by knorth55** [comment]
Fixed

**Comment by wkentaro** [comment]
LGTM

**Pull request merged**